src/neuroglancer/datasource/precomputed/frontend.ts

```diff
@@ -260,7 +263,7 @@ async function getMeshMetadata(
   try {
     metadata = await getJsonMetadata(chunkManager, url);
   } catch (e) {
-    if (e instanceof HttpError && e.status === 404) {
+    if (e instanceof HttpError && (e.status === 404 || e.status === 403)) {
```

Comment:
Just curious, what sort of serving setup gives 403?

Reply:
Regarding Google storage returning 403, does that happen when you grant access only via per-object ACLs, rather than per-bucket IAM permissions?

Reply:
"This bucket uses fine-grained access control, allowing you to specify access to individual objects. To control access uniformly at the bucket level, switch to uniform access control."

Honestly, it might be worth switching to uniform. The option didn't exist when we created some of them.

Reply:
You don't have to switch --- the bucket-level IAM permissions apply even if you don't switch the policy.  A user is granted access to an object if either the bucket-level IAM permissions or object-level ACL grants access.  I recommend not switching to uniform at the moment if you benefit from HTTP cache control headers, because there is currently a limitation in GCS such that switching to uniform access control also prevents caching.